drs/drs-assembly/subscribe-model.conf

```diff
@@ -593,19 +671,33 @@ Current position of the cold stop. Contains the following:
 
 {
   subsystem =  IRIS
-  component =  ifs-scale-assembly.scaleMechanism
-  name =  position
+  component =  ifs-scale-assembly
+  name =  scaleMechanism
```

Comment:
i don't think this item is in the scale assembly.  should be just scale.  the name is the current attribute.  don't need to have IFSSCPOS.